packages/mermaid/src/diagrams/gantt/ganttDb.js

```diff
@@ -1,8 +1,8 @@
 import { sanitizeUrl } from '@braintree/sanitize-url';
-import dayjs from 'dayjs';
-import dayjsIsoWeek from 'dayjs/plugin/isoWeek.js';
-import dayjsCustomParseFormat from 'dayjs/plugin/customParseFormat.js';
-import dayjsAdvancedFormat from 'dayjs/plugin/advancedFormat.js';
+import dayjs from 'dayjs/esm';
+import dayjsIsoWeek from 'dayjs/esm/plugin/isoWeek';
+import dayjsCustomParseFormat from 'dayjs/esm/plugin/customParseFormat';
+import dayjsAdvancedFormat from 'dayjs/esm/plugin/advancedFormat';
```

Comment:
Can you add `.js`/`.mjs`/`/index.js` to these imports?

Our bundler (Vite) doesn't care about this, but some libraries (i.e. see [Node.JS | ESM](https://nodejs.org/api/esm.html#mandatory-file-extensions)) require file extensions when importing from a file.

Reply:
the imports have been update to also include the files and extensions.. 😎